app/views/admin/custom_fields/_new_option.haml

```diff
@@ -4,6 +4,7 @@
       - show_locales = available_locales.size > 1
       .col-8{:class => show_locales ? "" : "custom-field-column-without-bottom-margin"}
         = fields_for "custom_field[option_attributes][#{option_id}][]", option do |option_form|
+          = hidden_field_tag "custom_field[option_attributes][#{option_id}][id]", option_id
```

Comment:
This is added to make the it possible to use Entities.

Previously the format of the options was following:

```
{
  "123" => { sort_priority: 1, ... other option values },
  "234" => { sort_priority: 2, ... other option values },
}
```

That format made it difficult to use the ID, because it's only in the hash key. However, changing the format to this:

```
{
  "123" => { id: "123" sort_priority: 1, ... other option values },
  "234" => { id: "234", sort_priority: 2, ... other option values },
}
```

...makes it easier to access to the ID and now it's possible to define an Entity with `id` and `sort_priority`.